packages/openapi3/test/metadata.test.ts

```diff
@@ -0,0 +1,565 @@
+import { deepStrictEqual } from "assert";
```

Comment:
Should we also have some test at the rest library layer for the metadata functions?

Reply:
Yes, good point. I moved the functions late and didn't consider that. If it's OK with you I'd prefer to make that a follow-up issue.

Reply:
I will file an issue for this.

Reply:
https://github.com/microsoft/cadl/issues/1071